src/CoreBundle/CronJob/CronJobScheduler.php

```diff
@@ -103,6 +103,17 @@ public function calculateCurrentPlannedExecutionDate(CronJobScheduleDataModel $s
         return $plannedExecution;
     }
 
+    private function isLess(\DateInterval $a, \DateInterval  $b): bool
+    {
+        $now = $this->timeProvider->getDateTime();
```

Comment:
This seems rather magical/workaround-ish: if necessary a short comment on why would be nice.

Reply:
method has been removed